src/renderer/widgets/CreateWallet/model/flow-model.ts

```diff
@@ -411,6 +411,29 @@ sample({
   target: flowFinished,
 });
 
+sample({
+  clock: signModel.output.formSubmitted,
+  source: {
+    signatories: signatoryModel.$signatories,
+    contacts: contactModel.$contacts,
+  },
+  fn: ({ signatories, contacts }) => {
+    return signatories.slice(1).reduce<Contact[]>((acc, { address, name }) => {
```

Comment:
I suggest to rewrite using `for_or`
Also there is a validation in contact create form that forbids duplicated `accountId`, I assume we can't have duplicated contacts with the same `accountId`
Thus we can create a lookup set `contactsSet: Set<AccountId>` and use it instead of `.find`